src/lib_ccx/params.c

```diff
@@ -2572,6 +2582,12 @@ int parse_parameters (struct ccx_s_options *opt, int argc, char *argv[])
 	opt->enc_cfg.gui_mode_reports = opt->gui_mode_reports;
 	if(opt->enc_cfg.render_font==NULL)
 		opt->enc_cfg.render_font = DEFAULT_FONT_PATH;
+	if (opt->enc_cfg.render_font_italics == NULL)
+		opt->enc_cfg.render_font_italics = DEFAULT_FONT_PATH_ITALICS;
```

Comment:
This is always true since there's no play in which opt->enc_cfg.render_font_italics changes. For the standard font we do have the parameter --font, but there isn't an equivalent for --italics (or bold)

Reply:
I'll add the flags.